### DISCUSSION TITLE
Customize Select Hover/Focus Border Color with ConfigProvider?

### DISCUSSION THREAD

**Discussion by esu-skoopin** [ant-design/ant-design#50898] [Q&A] [answered]
Can I customize the hover and focus border color for `Select` with `ConfigProvider`? I know that it's possible for `Input` components (see below), but there don't seem to be equivalent props for `Select`.
```
<ConfigProvider
    theme={{
        components: {
            Input: {
                hoverBorderColor: '#d9d9d9',
                activeBorderColor: '#d9d9d9'
            }
    }}
>
```
Currently, I'm overriding the default style with the following CSS, but I'd like to hear from the community if there's a better way to do this.
```
.ant-select:hover .ant-select-selector, .ant-select-focused .ant-select-selector {
    border-color: #d9d9d9 !important;
}
```
Thanks in advance 🙏

**Comment by kiner-tang** [comment]
Maybe we can support these tokens in select. Let me have a try!

**Reply by kiner-tang** [reply, marked as answer]
- https://github.com/ant-design/ant-design/pull/50951
